### PR TITLE
Profiling: Add docs for Parca and Phlare

### PR DESCRIPTION
**Pull request by aocenas**
Docs for the data sources added [here](https://github.com/grafana/grafana/pull/57809)

TODO:
- [x] add images https://github.com/grafana/website/pull/10570
- [x] add feature flag disclaimer

### CODE REVIEW

File: docs/sources/datasources/phlare.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+aliases:
+- /docs/grafana/latest/datasources/phlare/
+- /docs/grafana/latest/features/datasources/phlare/
+  description: Horizontally-scalable, highly-available, multi-tenant continuous profiling aggregation system. OSS profiling solution from Grafana Labs.
+  keywords:
+- grafana
+- phlare
+- guide
+- profiling
+  title: Phlare
```
Comment (Eve832) [inline docs/sources/datasources/phlare.md:11]:
```suggestion
  title: Phlare data source
```

File: docs/sources/datasources/phlare.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  weight: 1150
+---
+
+# Phlare data source
+
+> **Note:** This feature is behind the `flameGraph` feature toggle.
+> You can enable feature toggles through configuration file or environment variables. See configuration [docs]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}) for details.
+> Grafana Cloud users can access this feature by [opening a support ticket in the Cloud Portal](https://grafana.com/profile/org#support).
+
+Grafana ships with built-in support for Phlare, a horizontally scalable, highly-available, multi-tenant, OSS, continuous profiling aggregation system from Grafana Labs. Add it as a data source, and you are ready to query your profiles in [Explore]({{< relref "../explore" >}}).
```
Comment (Eve832) [inline docs/sources/datasources/phlare.md:21]:
```suggestion
Grafana ships with built-in support for Phlare, a horizontally scalable, highly-available, multi-tenant, OSS, continuous profiling aggregation system from Grafana Labs. When you install the Phlare data source, you can query your profiles in [Explore]({{< relref "../explore" >}}).
```

File: docs/sources/datasources/phlare.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+## Add data source
```
Comment (Eve832) [inline docs/sources/datasources/phlare.md:23]: Same general comments as Parca

**Review by Eve832 — COMMENTED** [review]
Leaving comments and suggestions without blocking a merge

File: docs/sources/datasources/parca.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  weight: 1110
+---
+
+# Parca data source
+
+> **Note:** This feature is behind the `flameGraph` feature toggle.
+> You can enable feature toggles through configuration file or environment variables. See configuration [docs]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}) for details.
+> Grafana Cloud users can access this feature by [opening a support ticket in the Cloud Portal](https://grafana.com/profile/org#support).
+
+Grafana ships with built-in support for Parca, a continuous profiling OSS database for analysis of CPU and memory usage, down to the line number and throughout time. Add it as a data source, and you are ready to query your profiles in [Explore]({{< relref "../explore" >}}).
```
[the remark this answers is not in the document]
Comment (aocenas) [reply docs/sources/datasources/parca.md:20]: I think this makes sense as it is, both parca and phlare will be part of grafana so no need to install it.

File: docs/sources/datasources/phlare.md (hunk 2 above)
Comment (aocenas) [reply docs/sources/datasources/phlare.md:21]: Same as for Parca as this is in Grafana already no need to install it.

File: docs/sources/datasources/phlare.md (hunk 1 above)
Comment (aocenas) [reply docs/sources/datasources/phlare.md]: Actually did not change it after all. In https://grafana.com/docs/grafana/latest/datasources/ we show it on the left under Datasources header so it looks weird to repeat that after each name. We have that right now only with influx not sure if accidental or not.

**Review by joey-grafana — APPROVED** [review]

**Pull request merged**